newscoop/src/Newscoop/NewscoopBundle/Resources/public/js/playlists/controllers/featured-articles.js

```diff
@@ -13,6 +13,7 @@ angular.module('playlistsApp').controller('FeaturedController', [
         Playlist
     ) {
 
+    var countDownTimeInSeconds = 11;
```

Comment:
I increased it to 11 seconds since I introduced "ok" button, 5 seconds was too short for showing revert popup. It counts from 11 so on 0 seconds it can close the popup and stop counting at 1.